paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt

```diff
@@ -352,20 +352,22 @@ class PaymentSheetPlaygroundActivity : AppCompatActivity() {
 
     override fun onResume() {
         super.onResume()
-        val savedToggles = viewModel.getSavedToggleState()
-        setToggles(
-            initialization = savedToggles.initialization,
-            customer = savedToggles.customer,
-            link = savedToggles.link,
-            googlePay = savedToggles.googlePay,
-            currency = savedToggles.currency,
-            merchantCountryCode = savedToggles.merchantCountryCode,
-            mode = savedToggles.mode,
-            shippingAddress = savedToggles.shippingAddress,
-            setAutomaticPaymentMethods = savedToggles.setAutomaticPaymentMethods,
-            setDelayedPaymentMethods = savedToggles.setDelayedPaymentMethods,
-            setDefaultBillingAddress = savedToggles.setDefaultBillingAddress
-        )
+        lifecycleScope.launch {
```

Comment:
Can we actually move this to `onCreate` instead of `onResume`? Having it in the latter causes it to run when we come back from PaymentSheet, which seems unnecessary to me.

Reply:
Makes sense to me. The behavior change still exists, but moving it to onCreate is logical, so I moved it.